agimus_demo_04_visual_servoing/scripts/reference_publisher

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
please use the `setup.cfg` just like in: https://github.com/agimus-project/agimus-demos/blob/humble-devel/agimus_demos_common/setup.cfg